src/components/DimensionMenu.js

```diff
@@ -248,7 +249,7 @@ export class DimensionMenu extends Component {
                     {i18n.t(
                         `${getAxisItemLabelPrefix(
                             isDimensionInLayout
-                        )} ${getAxisName(axisId)}`
```

Comment:
This should use `interpolation` instead of string template literals, even though we're not currently extracting the strings it's a good idea to fix it now.

Reply:
This `i18n.t` is also not going to do anything, so it should be removed.  Just note for the future that using a string template literal here will ask translators to translate your code, rather than a string with a variable, so [interpolation](https://www.i18next.com/translation-function/interpolation) is required when including variables in translated strings.

Reply:
Valid point. Removed i18n here